week-4/Homework/mandatory/2-writers.js

```diff
@@ -54,19 +54,58 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 
-
+writers.forEach((writer) =>
+  console.log(
+    `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`
+  )
+);
 
 /*
 Exercise 2:
   Only `console.log()` out the writers who are in their 40s (meaning between 40 and 49)
   and not alive anymore. Use the below sentence format:
   "Writer {firstName} {lastName} died at {age} years old."
 */
-
-
-
+function getNotAliveIn40s() {
+  return writers
+    .filter(
+      (writer) => writer.alive === false && writer.age >= 40 && writer.age < 50
+    )
+    .map((writer) =>
+      console.log(
+        "Writer " +
+          writer.firstName +
+          " " +
+          writer.lastName +
+          " died at " +
+          writer.age +
+          " years old."
+      )
+    );
+}

```

Comment:
Hi Leida, In this function, you could have used !writer.alive instead of writer.alive === false. 
For the second method, the instructor tells you the print out that writers information as highlighted.

Reply:
corrected